pkg/relayer/session/session.go

```diff
@@ -165,18 +166,21 @@ func (rs *relayerSessionsManager) mapBlockToSessionsToClaim(
 	defer rs.sessionsTreesMu.Unlock()
 
 	// Check if there are sessions that need to enter the claim/proof phase
```

Comment:
I think you should change this comment back. The PREVIOUS comment is the business logic. Inside of it, we add an explanation regarding the grace period.

It's similar to a database `GET`. You explain what the `GET` does, but not the "caching" that may or may not happen inside.